src/common/utils/excluded_apps.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include <shellapi.h>
+#include <vector>
+#include <string>
+
+inline bool detect_game_mode()
+{
+    QUERY_USER_NOTIFICATION_STATE notification_state;
+    if (SHQueryUserNotificationState(&notification_state) != S_OK)
+    {
+        return false;
+    }
+    return (notification_state == QUNS_RUNNING_D3D_FULL_SCREEN);
+}
```

Comment:
Why did you combine excluded apps and game mode? It's counterintuitive to search `detect_game_mode` in the `excluded_apps.h`.
IMO, let's keep them in separate files.

Reply:
Felt right at the time, since the idea of detecting game mode is usually to exclude those apps.
It's OK to separate those functions, though. I'll take care of it.
Thank you for the review.

Reply:
It's done. PTAL